.github/workflows/deployEABeta.yaml

```diff
@@ -3,7 +3,7 @@ concurrency: deploy-ea-beta
 on:
   workflow_dispatch:
   push:
-    branches: [ea-deploy]
+    branches: [wh-perf-2023-06]
```

Comment:
Also preemptively changing this to do some testing on the beta site

Reply:
Is it meant to be this branch, because you're missing `-settings` in the branch name if so?

Reply:
It's not meant to be this branch 👍